Source_Projects/FdvsFileInformation/DeliveryFileInfoModel.cs

```
@@ -18,12 +19,19 @@ public class DeliveryFileInfoModel
 
     public DeliveryFileInfoModel(string filePath)
     {
-      SetFileInfoValues();
+      SetFileInfoValues(filePath);
     }
 
-    private void SetFileInfoValues()
+    private void SetFileInfoValues(string filePath)
     {
-      //TODO - SetFileInfoValues()
+      var fileInfo = new FileInfo(filePath);
+      if (fileInfo.Exists)
```

fail early ? bättre kolla att den existerar redan i ctor?